### ISSUE TITLE
test-batch 是只能设置为1 吗 ？

### ISSUE THREAD

**Issue by 13121283123** [jiyanggao/Video-Person-ReID#26]
请问test-batch 是只能设置为1 吗 ？ 这样测试的时候太慢了，如果test-batch设置为别的数值的话，
queryloader = DataLoader(
        VideoDataset(dataset.query, seq_len=args.seq_len, sample='dense', transform=transform_test),
        batch_size=args.test_batch, shuffle=False, num_workers=args.workers,
        pin_memory=pin_memory, drop_last=False,
    )
就会报错，stack expects each tensor to be equal size，but got [10,4,3,256,128] at entry 0 and [25,4,3,256,128] at entry 1.
只有test-batch 为1的时候才能运行成功，请问大佬，有什么方法可以使test-batch 设置的大一些呢
